tv/tv.cpp

```diff
@@ -537,12 +539,16 @@ struct TVInitPass : public llvm::PassInfoMixin<TVInitPass> {
 
 struct TVFinalizePass : public llvm::PassInfoMixin<TVFinalizePass> {
   static bool finalized;
-  llvm::PreservedAnalyses run(llvm::Function &F,
-                              llvm::FunctionAnalysisManager &FAM) {
```

Comment:
TVFinalizePass is changed to module-level pass from function-level pass because there is no reason for it to be function level